frame/ddc-metrics-offchain-worker/src/lib.rs

```diff
@@ -417,25 +422,27 @@ impl<T: Trait> Module<T> {
         info!("[OCW] Using Contract Address: {:?}", contract_id);
 
         for one_metric in metrics.iter() {
-            if one_metric.bytes == 0 && one_metric.requests == 0 {
+            if one_metric.storageBytes == 0 && one_metric.wcuUsed == 0 && one_metric.rcuUsed == 0 {
```

Comment:
Since the function `report_metrics_ddn` also means "node is online", it should call it even with 0 metrics.

Reply:
@doasync Is that correct?

(this is marked as outdated but it’s still there)

Reply:
[done](https://github.com/Cerebellum-Network/pos-network-node/pull/66/files#diff-2cf146c84dbe7f3d47b44a1345d0bed15228bcd3aecb643c6c0abd42f7803a80L420-L423) for send_metrics_ddn_to_sc